docs/sphinx/source/installation.rst

```diff
@@ -34,49 +33,153 @@ There are many ways to install Python on your system, but the Anaconda
 Scientific Python distribution provides by far the easiest way for new
 users to get started. Anaconda includes all of the popular libraries
 that you'll need for pvlib, including Pandas, NumPy, and SciPy.
-"Anaconda installs cleanly into a single directory, does not require
-Administrator or root privileges, does not affect other Python installs
-on your system, or interfere with OSX Frameworks." -Anaconda
-Documentation.
 
-#. Install the full Anaconda Scientific Python distribution available
-   `here <https://store.continuum.io/cshop/anaconda/>`_
-#. Install pvlib: ``conda install -c pvlib pvlib``
+    Anaconda installs cleanly into a single directory, does not require
+    Administrator or root privileges, does not affect other Python installs
```

Comment:
Thanks for this hint. It appears this has improved in the last months! I was not able to install as non-admin before.